packages/mongo-nav/README.md

```diff
@@ -25,8 +25,8 @@
 | `mode`                     | `production` or `dev`                                                           | Describes what environment the component is being used in, defaults to `production`                                                                                                                                                              | `production`                                            |
 | `onOrganizationChange`     | `(_value: string, _event: React.ChangeEvent) => void`                           | Callback invoked when user types into organization picker. Receives value of input as first argument and ChangeEvent as the second.                                                                                                              |                                                         |
 | `onProjectChange`          | `(_value: string, _event: React.ChangeEvent) => void`                           | Callback invoked when user types into project picker. Receives value of input as first argument and ChangeEvent as the second.                                                                                                                   |                                                         |
-| `constructOrganizationURL` | `(orgId) => string`                                                             | Function to determine destination URL when user selects an organization from the organization picker, see also `hosts`                                                                                                                           | `(orgId) => '${hosts.cloud}/v2#/org/${orgId}/projects'` |
-| `constructProjectURL`      | `(orgId, projId) => string`                                                     | Function to determine destination URL when user selects a project from the project picker, see also `hosts`                                                                                                                                      | `(orgId, projId) => '${hosts.cloud}/v2#/${projectId}'`  |
+| `constructOrganizationURL` | `(Organization) => string`                                                      | Function to determine destination URL when user selects an organization from the organization picker, see also `hosts`                                                                                                                           | `(orgId) => '${hosts.cloud}/v2#/org/${orgId}/projects'` |
```

Comment:
For the readme, if we refer to an interface we should define the interface either inline here (if it is short), or below the main block of props.

Reply:
The default is also now incorrect, since the argument is the entire object not just the id